src/client/opamAuxCommands.mli

```diff
@@ -87,8 +87,9 @@ val simulate_autopin:
 (** Scans for package definition files in a directory, and selects a compiler
     that is compatible with them from the configured default compiler list, or
     that is unambiguously selected by the package definitions.
-    Returns the corresponding atoms. If no compiler matches, prints a
-    warning, and returns the empty list after user confirmation. *)
+    Returns a couple: the corresponding atoms and a boolean that is true if the
```

Comment:
```suggestion
    Returns a pair: the corresponding atoms and a boolean that is true if the
```
I can't remember how many times my professor corrected me on this one when I studied in England ;)
Otherwise all good to merge, thanks!